modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
+
+----
+
+==== Example 3: Publish an asset regardless of the content
+
+If you want to publish a new version of the asset in Exchange regardless of the content, you should use the `--force-publish` option. 
+
+This example shows the `api-catalog publish-asset` command with both of these options. This example assumes that authentication credentials are set in environment variables.
+
+----
+> api-catalog publish-asset -a https://anypoint.mulesoft.com --descriptor-file=mydescriptor.yaml -o 1234abc-b34a-7cd3-4s32-12345abc2345 --force-publish 
+
+----
+
+==== Example 4: Publish an asset using trigger conditions 
+
+Custom trigger conditions enable you to further control what is published when the API Catalog CLI command runs. 
+
+This example shows `api-catalog publish-asset` command options that will publish only the APIs that are identified in the `triggerConditions` section of a descriptor file called `mydescriptor.yaml`.  
+
+*Command to publish:*
+
+----
+> api-catalog publish-asset -a https://anypoint.mulesoft.com -d mydescriptor.yaml -o 1234abc-b34a-7cd3-4s32-12345abc2345 -u myAnyPtAccount -p myPwd@4!myacct --trigger=ref:main --trigger=tag:migrate --trigger=status:complete
+
+----
+
+*Section in the specified descriptor file:*
+
+---
+ 
+ triggerConditions:
+  ref:
+   - main
+  tags:
+   - migrate
+   - republish
+  status:
+   - complete
+   - ready
+
+---
+
+==== Example 5: Publish Using a Jenkins Script 
+
+This example shows a Jenkins script that runs the `api-catalog publish-asset` command.
+
+[source,java]
+----
+pipeline {
+    agent any
+    environment {
+        ANYPOINT_ORG_ID = '1234abc-b34a-7cd3-4s32-12345abc2345'
+        CATALOG_DESCRIPTOR = './descriptor.yaml' <1>
+     }   
+     stages {
+        stage('git checkout') {
+            steps {
+                git url: 'git@github.com:mygitlocation/api-catalog-cli.git'
+                // additional checkout tasks
+            }
+        }
+        stage('Build Artifacts') {    
+            steps {
+                 // building
+            }
+        }
+        stage('API Cataloging') {    
+            steps {
+                 withCredentials([
+                    usernamePassword(credentialsId: 'my-anypoint-creds',
+                    usernameVariable: 'ANYPOINT_USERNAME',
+                    passwordVariable: 'ANYPOINT_PASSWORD')
+                ]) { 
+                    sh 'api-catalog-cli publish-asset -d $CATALOG_DESCRIPTOR -o $ANYPOINT_ORG_ID -a https://anypoint.mulesoft.com/ --trigger=branch:main' <2>
+                }
+            }
+        }
+         stage('Deploy') {    
+            steps {
+                // Any deployment tasks to be performed, here.
+            }
+        }
+    }
+----
+<1> Defines the organization and descriptor file values
+<2> Runs the `api-catalog publish-asset` command to publish APIs found in the `main` Github branch
+
+== Authentication
```

Comment:
Does this belong with Permissions?
This topic is not accounted for in the intro list of major subtopics.

Reply:
This is from a longer section in the Anypoint CLI doc, and will likely be cross-referenced in the GA doc. I couldn't cross-reference it yet since a) these commands don't support all of the authentication methods yet and b) the information in the Anypoint CLI doc is in the same page with the intro and installation doc for Anypoint CLI.

Reply:
It doesn't belong with permissions. It's about different ways of logging in through the command line.